.github/workflows/build_site.yml

```diff
@@ -0,0 +1,43 @@
+name: Build website
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  Build-website:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2 # FIXME: add pip + latex + gems
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
```

Comment:
If we change the dependencies, then the new version will be installed as directed by the gemfile. This will copy the previous version there. In any case, we can always delete the cache.